base_environment.yml

```diff
@@ -1,110 +1,98 @@
-  - google-cloud-container=0.3.0=py37_0
-  - google-cloud-storage=1.23.0=py37_0
-  - holoviews
-  - h5netcdf
-  - icu
-  - iris
-  - jedi
-# need server proxy on workers if using remote scheduler
-  - jupyter-server-proxy=1.2.0=py_0
-  - kubernetes
-  - lapack
-  - lz4
-  - make=4.2.1=h14c3975_2004
-  - matplotlib=3.1.2=py37_1
-  - nc-time-axis=1.2.0=py_0
-  - ncurses=6.1=hf484d3e_1002
-  - netcdf-fortran
-  - netcdf4
-  - numba
-  - numcodecs
-  - pandas=0.25.3=py37hb3f55d8_0
-  # for geoviews
-  - phantomjs=2.1.1=1
-  - pip=19.3.1=py37_0
-  - plotly=4.3.0=py_0
-  - polyline=1.4.0=py_0
-  - pygeos=0.5=py37h5d51c17_1
-  - pyinterp=0.0.7=py37h97f2665_0
-  - pyshp=2.1.0=py_0
-  - python=3.7.3=h357f687_2 # pinkeep: python=3.7
-  - python-blosc
-  - python-snappy=0.5.4=py37hee44bf9_1
-  - pyviz_comms=0.7.2=py_0
-  - pyyaml=5.2=py37h516909a_0
-  - rasterio
-  - regionmask=0.4.0=py_0
-  - rtree=0.8.3=py37h7b0cdae_1003
-  - scikit-image=0.16.2=py37hb3f55d8_0
-  - scikit-learn=0.22=py37hcdab131_0
-  - scipy=1.3.2=py37h921218d_0
-  - seaborn=0.9.0=py_2
-  # for geoviews
-  - selenium=3.141.0=py37h516909a_1000
-  - shapely=1.6.4=py37h5d51c17_1007
-  - sparse=0.8.0=py_0
-  - statsmodels=0.10.2=py37hc1659b7_0
-  - tini=0.18.0=h14c3975_1001
-  - unzip=6.0=h516909a_0
-  - uritemplate=3.0.0=py_1
-  - xarray=0.14.1=py_0
-  - xesmf
-  - xgcm
-  - xhistogram=0.1.1=py_0
-  - xlrd=1.2.0=py_0
-  - xrft=0.2.0=py_0
-  - zarr=2.3.2=py37_0
-  - zeromq=4.3.2=he1b5a44_2
-  - zict=1.0.0=py_0
-  - pip:
-    - mapbox==0.18.0
-    - py-noaa==1.0
-    - sidecar==0.3.0
-    - climate-toolbox==0.1.5
-    - impactlab-tools==0.4.0
-    - parameterize-jobs==0.1.1
-    - git+https://github.com/rhodiumgroup/rhg_compute_tools.git@allow-remote-scheduler#egg=rhg_compute_tools
-    - git+https://github.com/NCAR/intake-esm.git#egg=intake_esm
-# need to install from master until 0.10.1
-# due to handling of remote scheduler
-# (we also should at some point switch to dask-gateway instead of dask-kubernetes)
-    - git+https://github.com/dask/dask-kubernetes.git#egg=dask_kubernetes
+- beautifulsoup4=4.9
+- bottleneck=1.3
+- boto3=1.13
+- bqplot=0.12
+- bumpversion=0.5
+- cachecontrol=0.12
+- cartopy=0.17
+- cftime=1.1
+- click=7.1
+- compilers=1.0
+- dask=2.17
+- dask-glm=0.2
+- dask-kubernetes=0.10
+- dask-ml=1.5
+- datashader=0.11
+- descartes=1.1
+- distributed=2.17
+- dropbox=9.4
+- esmpy=7.1
+- fastparquet=0.4
+- fiona=1.8
+- fusepy=3.0
+- gcc_linux-64=7.5
+- gcsfs=0.6
+- gdal=3.0
+- geoalchemy2=0.6
+- geopandas=0.7
+- geopy=1.22
+- geotiff=1.5
+- geoviews=1.8
+- git=2.26
+- gitpython=3.1
+- google-cloud-container=0.3
+- google-cloud-storage=1.28
+- holoviews=1.13
+- h5netcdf=0.8
+- icu=58.2
+- intake-esm=2020.5
+- iris=2.4
+- jedi=0.17
+- jupyter-server-proxy=1.5
+- kubernetes=1.18
+- lapack=3.6
+- lz4=3.0
+- make=4.3
+- matplotlib=3.1
+- nc-time-axis=1.2
+- ncurses=6.1
+- netcdf-fortran=4.4
+- netcdf4=1.5
+- numba=0.48
+- numcodecs=0.6
+- pandas=1.0
+- phantomjs=2.1
```

Comment:
can we add back in the comment that this package (and selenium) are optional dependencies of geoviews (just so we don't accidentally delete in the future). Can you also add pillow (another optional dependency of geoviews)